src/domains/gps-common/env/weu-prod/terraform.tfvars

```diff
@@ -134,9 +133,20 @@ reporting_storage_account = {
   backup_retention           = 30
 }
 
-
 geo_replica_enabled                = true
 location_replica                   = "northeurope"
 location_replica_short             = "neu"
 geo_replica_cidr_subnet_postgresql = ["10.2.141.0/24"]
 postgresql_sku_name                = "GP_Gen5_2"
+
+# GPD Storage Account SFTP
+gpd_account_replication_type                                       = "GRS"
+cidr_subnet_gpd_storage_account                                    = ["10.1.152.16/29"]
+gpd_sa_blob_versioning_enabled                                     = true
```

Comment:
@AngeloCaporaso should be set to `false` for https://learn.microsoft.com/en-us/azure/storage/blobs/secure-file-transfer-protocol-known-issues